src/main/java/org/breedinginsight/services/validators/TraitValidatorService.java

```diff
@@ -195,11 +202,8 @@ public ValidationErrors checkDuplicateTraitsInFile(List<Trait> traits, TraitVali
         Map<String, List<Integer>> namesMap = new HashMap<>();
         for (Integer i = 0; i < traits.size(); i++) {
             Trait trait = traits.get(i);
-            if (trait.getTraitName() != null &&
-                    trait.getScale() != null && trait.getScale().getScaleName() != null
-            ){
-                String key = String.format("%s-%s", trait.getTraitName().toLowerCase(),
-                        trait.getScale().getScaleName().toLowerCase());
+            if (trait.getTraitName() != null){
+                String key = String.format("%s", trait.getTraitName().toLowerCase());
```

Comment:
The `String.format("%s"` is redundant, and can just be `String key = trait.getTraitName().toLowerCase();`

Reply:
Changed and pushed